src/ec/curve25519/ed25519.rs

```diff
@@ -228,6 +228,13 @@ impl<'a> Ed25519KeyPairComponents {
         Self::from_seed_unchecked(seed)
     }
 
+    /// Constructs an `Ed25519KeyPair` from these components.
+    pub fn to_key_pair(&self) -> Result<Ed25519KeyPair, error::Unspecified> {
```

Comment:
I specifically want to avoid having a `to_key_pair` function here, so let's remove this commit. Users can implement this logic themselves if they want.

Reply:
> I specifically want to avoid having a to_key_pair function here, so let's remove this commit. Users can implement this logic themselves if they want.

Never mind this, I misunderstood and I've changed my mind. Ignore the previous comment.